src/main/java/org/opensearch/security/privileges/PrivilegesEvaluator.java

```diff
@@ -103,8 +103,8 @@ public class PrivilegesEvaluator {
 
     private static final WildcardMatcher ACTION_MATCHER = WildcardMatcher.from("indices:data/read/*search*");
 
-    private static final Pattern DNFOF_PATTERNS = Pattern.compile(
-        "indices:(data/read/.*|(admin/(mappings/fields/get.*|shards/search_shards|resolve/index)))"
+    public static final Pattern DNFOF_PATTERNS = Pattern.compile(
+        "indices:(data/read/.*|(admin/(mappings/fields/get.*|shards/search_shards|resolve/index))|(monitor/((settings/get)|(stats))))"
```

Comment:
I'm not familiar with this option - I included all index permissions in a unit test so that should be pretty clear/tell us if something in the regex changed in an unexpected way, but this may be more clear. Any advantage of WildcardMatcher over checking inclusion in a set/something similar?

Reply:
I ended up turning it into a function like the isClusterPerm, I think that is relatively clean, maintainable, and consistent with the code base.